app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: 'Book was successfully created.' }
+        format.html { redirect_to book_url(@book), notice: t('books.index.notice.create') }
```

Comment:
ためしに

```suggestion
        format.html { redirect_to book_url(@book), notice: t('.notice.create') }
```

としてみてください。
エラーメッセージを見ると、Railsが標準で探しに行く翻訳のキー名がわかるはずなので、そのキー名を使ってみましょう。

Reply:
> Railsが標準で探しに行く翻訳のキー名がわかるはずなので、そのキー名を使ってみましょう。

ありがとうございます！試したところ、Railsが標準で探しに行く翻訳キー名が分かりました！

![CleanShot 2025-05-31 at 22 19 14](https://github.com/user-attachments/assets/2166b7a1-c314-4224-b431-937f6bce96f6)


![CleanShot 2025-05-31 at 22 20 14](https://github.com/user-attachments/assets/77bdf2d0-438d-4cd8-a6af-61c747d792a7)